assets/options.html

```diff
@@ -3,23 +3,31 @@
2);box-shadow:none}.btn-primary{color:#fff;background-color:#28a745;background-image:-webkit-linear-gradient(top, #34d058 0%, #28a745 90%);background-image:linear-gradient(-180deg, #34d058 0%, #28a745 90%)}.btn-primary:focus,.btn-primary.focus{box-shadow:0 0 0 0.2em rgba(52,208,88,0.3)}.btn-primary:hover,.btn-primary.hover{background-color:#269f42;background-image:-webkit-linear-gradient(top, #2fcb53 0%, #269f42 90%);background-image:linear-gradient(-180deg, #2fcb53 0%, #269f42 90%);background-position:0 -0.5em;border-color:rgba(27,31,35,0.5)}.btn-primary:active,.btn-primary.selected{background-color:#279f43;background-image:none;border-color:rgba(27,31,35,0.5);box-shadow:inset 0 0.15em 0.3em rgba(27,31,35,0.15)}.btn-primary:disabled,.btn-primary.disabled{color:rgba(255,255,255,0.75);background-color:#94d3a2;background-image:none;border-color:rgba(27,31,35,0.2);box-shadow:none}.btn-primary .counter{color:#29b249;background-color:#fff}.btn-purple{color:#fff;background-color:#643ab0;background-image:-webkit-linear-gradient(top, #7e55c7 0%, #643ab0 90%);background-image:linear-gradient(-180deg, #7e55c7 0%, #643ab0 90%)}.btn-purple:focus,.btn-purple.focus{box-shadow:0 0 0 0.2em rgba(126,85,199,0.3)}.btn-purple:hover,.btn-purple.hover{background-color:#5f37a8;background-image:-webkit-linear-gradient(top, #784ec5 0%, #5f37a8 90%);background-image:linear-gradient(-180deg, #784ec5 0%, #5f37a8 90%);background-position:0 -0.5em;border-color:rgba(27,31,35,0.5)}.btn-purple:active,.btn-purple.selected{background-color:#613ca4;background-image:none;border-color:rgba(27,31,35,0.5);box-shadow:inset 0 0.15em 0.3em rgba(27,31,35,0.15)}.btn-purple:disabled,.btn-purple.disabled{color:rgba(255,255,255,0.75);background-color:#b19cd7;background-image:none;border-color:rgba(27,31,35,0.2);box-shadow:none}.btn-purple .counter{color:#683cb8;background-color:#fff}.btn-blue{color:#fff;background-color:#0361cc;background-image:-webkit-linear-gradient(top, #0679fc 0%, #0361cc 90%);background-image:linear-gradient(-180deg, #0679fc 0%, #0361cc 90%)}.btn-blue:focus,.btn-blue.focus{box-shadow:0 0 0 0.2em rgba(6,121,252,0.3)}.btn-blue:hover,.btn-blue.hover{background-color:#035cc2;background-image:-webkit-linear-gradient(top, #0374f4 0%, #035cc2 90%);background-image:linear-gradient(-180deg, #0374f4 0%, #035cc2 90%);background-position:0 -0.5em;border-color:rgba(27,31,35,0.5)}.btn-blue:active,.btn-blue.selected{background-color:#045cc1;background-image:none;border-color:rgba(27,31,35,0.5);box-shadow:inset 0 0.15em 0.3em rgba(27,31,35,0.15)}.btn-blue:disabled,.btn-blue.disabled{color:rgba(255,255,255,0.75);background-color:#81b0e5;background-image:none;border-color:rgba(27,31,35,0.2);box-shadow:none}.btn-blue .counter{color:#0366d6;background-color:#fff}.btn-danger{color:#cb2431;background-color:#fafbfc;background-image:-webkit-linear-gradient(top, #fafbfc 0%, #eff3f6 90%);background-image:linear-gradient(-180deg, #fafbfc 0%, #eff3f6 90%)}.btn-danger:focus{box-shadow:0 0 0 0.2em rgb
```

Comment:
Instead of pasting all this stuff in, what if we just add a couple link elements to the https://unpkg.com versions of the relevant files in the published npm packages? We can use [subresource integrity](https://www.srihash.org/) to verify that they do not change. This would look like:

```html
<link rel="stylesheet" href="https://unpkg.com/primer-css@5.0.0/build/build.css" integrity="sha384-Bg/C6ZDEvQjQtboCzdJbsH6QCir2Z4k57aGBdganyzSMwOTKqQJUjU4y6IN/Su6G" crossorigin="anonymous">
<link rel="stylesheet" href="https://unpkg.com/primer-forms@1.0.6/build/build.css" integrity="sha384-V1f3Tj00bC3QZNxrIbWHpggMZpFzizEQoTVf7lDfJ1ktkuokeoy/yntf187gJXo4" crossorigin="anonymous">
```